services/metadactyl-clj/src/metadactyl/service/apps.clj

```diff
@@ -238,7 +239,9 @@
    (log/info "synchronizing the job status for" id)
    (transaction (jobs/sync-job-status (get-apps-client-for-username username) job))
    (catch Object _
-     (log/error (:throwable &throw-context) "unable to sync the job status for job" id))))
+     (log/error (:throwable &throw-context) "unable to sync the job status for job" id))
+   (finally
+     (clear-ext-svc-tag!))))
```

Comment:
If this is already inside a logging context, `with-logging-context` already does its clear/replace thing in a `finally` block; if it's not in a logging context, it's essentially in a root logging context (of a sort) and will get cleared at the end of the request.